packaging/installer/README.md

```diff
@@ -165,3 +165,63 @@ the community helps fix any bugs that might have been introduced in previous rel
 -   Protect yourself from the rare instance when major bugs slip through our testing and negatively affect a Netdata
     installation
 -   Retain more control over the Netdata version you use
+
+## Installation notes and known issues
+
+We are tracking a few issues related to installation and packaging.
+
+### Failure to claim/connect to Netdata Cloud
+
+If you're running an older Linux distribution or one that has reached EOL, such as Ubuntu 14.04 LTS, Debian 8, or CentOS
+6/7, your Agent may not be able to connect to Cloud due to an old version of OpenSSL that is incompatible with Netdata
+Cloud.
+
+We recommend using the [static build installation](/packaging/installer/methods/kickstart-64.md) to avoid this issue.
+You can, however, use the workaround below.
+
+#### OpenSSL workaround
+
+Instead of using the static build, you can also reconfigure OpenSSL to allow self-signed certificates and skip hostname
```

Comment:
@amoss / @netdata/infra Please correct me if I'm wrong here, but I don't think "self-signed" certs are really relevant here. It is simply that we are disabling hostname verification.

Reply:
In fact I'm kind of curious; @amoss Do we require all 3 bit flags here to make this work on systems with an older openssl that do not have `X509_VERIFY_PARAM_set1_host` feature

```#!c
LCCSCF_ALLOW_SELFSIGNED | LCCSCF_SKIP_SERVER_CERT_HOSTNAME_CHECK | LCCSCF_ALLOW_INSECURE
```

Is it not enough to set `LCCSCF_SKIP_SERVER_CERT_HOSTNAME_CHECK` option?

Reply:
@prologic I concur with both you comments:

> I don't think "self-signed" certs are really relevant here. It is simply that we are disabling hostname verification.

￼I thought so too.

> Is it not enough to set LCCSCF_SKIP_SERVER_CERT_HOSTNAME_CHECK option?

My understanding is that it should be enough.

Reply:
I think that it would be enough.

Reply:
Then I would prefer we remove the wording around "self-signed" and "work-around" and state what's actually going on here.

Reply:
The self-signed stuff was my bad—I misinterpreted what the option was doing.